app/models/totp_form.rb

```diff
@@ -0,0 +1,4 @@
+class TotpForm
```

Comment:
Is this used anywhere?

Reply:
Removed... sorry this was left over from our first attempt to make this work.